config.md

```diff
@@ -356,7 +356,8 @@ Runtime implementations MAY support any valid values for platform-specific field
 
 ## <a name="configHooks" />Hooks
 
-Hooks allow for the configuration of custom actions related to the [lifecycle](runtime.md#lifecycle) of the container.
+Hooks allow for the configuration of custom actions related to the [lifecycle](runtime.md#lifecycle) of the container if supported by the platform.
+On Linux, they are run after the container namespaces are created.
```

Comment:
Not all hooks are run after creation.  In fact, there are currently no `postcreate` hooks (`prestart` hooks are called “[after the `start` operation is called but before the user-specified program command is executed][1]”.  I suggest not adding this line (and removing it from the Go comment).

[1]: https://github.com/opencontainers/runtime-spec/blob/2ec18c02474dd49ce20aa30236959a8a80763042/config.md#prestart